db/seeds/04_menu_items.sql

```diff
@@ -0,0 +1,12 @@
+INSERT INTO menu_items (restaurant_id, name, description, image_url, price, prep_time, menu_item_category_ID)
+VALUES(1, 'Henryburger', 'a burger with cheese, lettuce, tomatoes', 'docs/menu_items_images/cheeseburger.jpg', 9, 5, 1),
+(1, 'Henryveggie', 'vegan friendly burger with cheese, lettuce, tomatoes', 'docs/menu_items_images/veggieburger.jpg', 8, 5, 1),
+(1, 'Henryfries', 'Henry approved french fries', 'docs/menu_items_images/fries.jpg', 3, 4, 2),
+(1, 'Poutine', 'french fries with cheese curds and gravy', 'docs/menu_items_images/poutine.jpg', 8, 6, 2 ),
+(1, 'House salad', 'lettuce and veggies - not Henry approved', 'docs/menu_items_images/salad.jpg', 7, 3, 2),
+(1, 'soda', 'bubbly and refreshing - not sponsored', 'docs/menu_items_images/soda.jpg', 2, 1, 3),
+(1, 'beer', 'bubbly and intoxicating', 'docs/menu_items_images/beer.jpg', 5, 1, 3),
+(1, 'bottled water', 'bottled fresh from the tap for your convenience', 'docs/menu_items_images/water.jpg', 3, 1, 3),
+(1, 'cupcake', 'end your meal with a cake that fits in a cup', 'docs/menu_items_images/cupcake.jpg',5, 1, 4),
+(1, 'doughnut', 'delicioius nut free doughnuts', 'docs/menu_items_images/doughnut.jpg', 4, 1, 4);
```

Comment:
Should we have price as cents and convert them when showing on the page? Like the 8 dollars burger would be 800.